content/en/blog/_posts/2022-11-28-registry-k8s-io-change.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: blog
+title: "registry.k8s.io: faster, cheaper and Generally Available (GA)"
+date: 2022-11-28
+slug: registry-k8s-io-faster-cheaper-ga
+---
+# registry.k8s.io: faster, cheaper and Generally Available (GA)
+
+Starting with Kubernetes 1.25, our container image registry has changed from k8s.gcr.io to [registry.k8s.io](https://registry.k8s.io). This new registry spreads the load across multiple Cloud Providers & Regions, functioning as a sort of content delivery network (CDN) for Kubernetes container images. This change reduces the project’s reliance on a single entity and provides a faster download experience for a large number of users.
```

Comment:
[[Preview](https://deploy-preview-38059--kubernetes-io-main-staging.netlify.app/blog/2022/11/28/registry-k8s-io-faster-cheaper-ga/)]
![image](https://user-images.githubusercontent.com/46767780/203881971-86bf562e-72ab-4103-ba99-57e1393ddca1.png)

In the rendered version, the title is duplicated; the first one is from the front-matter, and the second one is from the H1 heading.
So I suggest to remove the latter one.
And let's add the 'Author' marker. 😊

```suggestion
---

**Author**: Frederico Muñoz (SAS)

Starting with Kubernetes 1.25, our container image registry has changed from k8s.gcr.io to [registry.k8s.io](https://registry.k8s.io). This new registry spreads the load across multiple Cloud Providers & Regions, functioning as a sort of content delivery network (CDN) for Kubernetes container images. This change reduces the project’s reliance on a single entity and provides a faster download experience for a large number of users.
```

Reply:
I think we'll need to Adolfo's name too since he has co-authored the blog in addition to the above.

Reply:
@jihoon-seo @jihoon-seo actually, only Adolfo's name should be there, mine shouldn't be references at all. I hadn't added it because I was going to check how @puerco preferred to be be styled (with company name or without, with "Puerco" also mentioned, etc) but didn't wanted to delay the  possible reviews.
That said, I'll add something as a starting point to make it clear who is the author and change it later.

Reply:
@divya-mohan0209 @jihoon-seo actually, only @puerco 's name shjould be there, I didn't add it because I was going to check how he preferred it to be displayed. I'll add something right now though.